packages/tasks/src/model-libraries-downloads.ts

```diff
@@ -6,15 +6,13 @@
  * Read this doc about download stats on the Hub:
  *
  * https://huggingface.co/docs/hub/models-download-stats
- *
+ * Available fields:
+ *  - path: the complete file path (relative) (e.g: "prefix/file.extension")
+ *  - path_prefix: the prefix of the file path (e.g: "prefix/", empty if no prefix)
+ *  - path_extension: the extension of the file path (e.g: "extension")
+ *  - path_filename: the name of the file path (e.g: "file")
  * see also:
- * https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html
+ * https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html
  */
 
-export type ElasticBoolQueryFilter =
-	// match a single filename
-	| { term?: { path: string } }
-	// match multiple possible filenames
-	| { terms?: { path: string[] } }
-	// match a wildcard
-	| { wildcard?: { path: string } };
+export type ElasticSearchQuery = string;
```

Comment:
i haven't read the linked internal PR yet but is the move to a string (vs. an object) a good idea?

Reply:
or is it a query language natively supported by Elastic?

Reply:
| is it a query language natively supported by Elastic?

Though the good operation yes (`query_string`)

I think it'll simplify how to contribute because DSL is not common vs a basic boolean language